### PR TITLE
Update QuickStart iOS Swift Authorization to use idToken custom claims

### PR DESCRIPTION
**Pull request by cocojoe**
Sample PR https://github.com/auth0-samples/auth0-ios-swift-sample/pull/43

### CODE REVIEW

File: articles/quickstart/native/ios-swift/05-authorization.md [hunk 1]
```diff
@@ -17,55 +17,59 @@ budicon: 500
...
+}
 ```
-if (user.email.indexOf('@example.com') > -1)
-```
+
+The rule will run every time a user attempts to authenticate. If the user has a valid `email` and the email's domain is `admin.com`, the user will be granted the `user` and `admin` roles. On any other case, the user will be granted a simple `user` role. This new claim is saved in the *id_token* under the `https://access.control/roles` name, which you will have to check on your app.
 
 ::: note
-You can set roles other than `admin` and `user` or customize the rule as needed.
+Feel free to customize the conditions and the roles to grant on each of them. You can define as many roles as your project requires, beyond `admin` and `user`. There are some restrictions on the naming of the claims detailed in the [rules documentation](/rules/#hello-world).
```
Comment (lbalmaceda) [inline articles/quickstart/native/ios-swift/05-authorization.md]: URL is `/rules#hello-world`

File: articles/quickstart/native/ios-swift/05-authorization.md [hunk 2]
```diff
@@ -17,55 +17,59 @@ budicon: 500
...
-        }
-}
+guard
+    let idToken = self.keychain.string(forKey: "id_token"),
+    let jwt = try? decode(jwt: idToken),
+    let roles = jwt.claim(name: "https://access.control/roles").array
+    else { // Couldn't retrieve claim }
+
+    if roles.contains("admin") {
+        // Authorised
```
Comment (lbalmaceda) [inline articles/quickstart/native/ios-swift/05-authorization.md]: Does the sample do something when these criteria are met? like allowing the user to access a different screen or something?

**Review by lbalmaceda — CHANGES_REQUESTED** [review]

File: articles/quickstart/native/ios-swift/05-authorization.md (hunk 1 above)
Comment (cocojoe) [reply articles/quickstart/native/ios-swift/05-authorization.md]: Either works 😄

File: articles/quickstart/native/ios-swift/05-authorization.md (hunk 2 above)
Comment (cocojoe) [reply articles/quickstart/native/ios-swift/05-authorization.md]: Yes, I could add that in comments

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**